packages/docs/src/components/documentation.tsx

```diff
@@ -166,6 +169,19 @@ export class Documentation extends React.PureComponent<IDocumentationProps, IDoc
         Utils.safeInvoke(this.props.onComponentUpdate, activePageId);
     }
 
+    private updateTitle(){
+        // Update the page <title> when the hash changes
+        let title:string;
+        if(this.state.activePageId === 'blueprint' && this.state.activeSectionId === 'blueprint'){
+            const indexTitle: string = 'Blueprint - Documentation';
```

Comment:
No need to specify the type explicitly here. TS knows it's a `string`.

also, double quotes